theano/compile/monitormode.py

```diff
@@ -82,10 +92,24 @@ def requiring(self, *tags):
         ret.post_func = self.post_func
         return ret
 
+    def clone(self, opt_kwargs=None, **kwargs):
+        """
+        Create a new instance of this Mode.
+
+        Keyword arguments can be provided for the linker,
+        but they will be ignored, because ProfileMode needs
```

Comment:
the param opt_kwargs is named link_kwargs in Mode. Can you make that consistent? Also kwargs isn't used in both class.

Can you use in both class just an link_kwargs parameter?